test/index.js

```diff
@@ -128,6 +128,20 @@ test('HashBase#update', function (t) {
     t.end()
   })
 
+  t.test('handle UInt16Array', function (t) {
+    beforeEach(t, createHashBase)
+
+    if (ArrayBuffer.isView && (Buffer.prototype instanceof Uint8Array || Buffer.TYPED_ARRAY_SUPPORT)) {
```

Comment:
these checks can go in a { skip: } options arg to the test. Often i do `{ skip: condition && 'explanation' }` and it displays nicely.